features/system_specs/system_specs.feature

```diff
@@ -0,0 +1,35 @@
+Feature: System spec
+
+    System specs are RSpec's wrapper around Rails' own
+    [system tests](http://guides.rubyonrails.org/testing.html#system-testing).
+    We encourage you to familiarse yourself with their documentation.
+
+    RSpec **does not** use your `ApplicationSystemTestCase` helper. Instead it uses
+    the default `driven_by(:selenium)` from Rails. If you want to override this
```

Comment:
Strongly disagree, selenium is the default in Rails and so I think it's better to mirror that default.